lib/davinci_dtr_test_kit/payer_server_groups/adaptive_next_questionnaire_extensions_test.rb

```diff
@@ -9,9 +9,12 @@ class PayerAdaptiveNexQuestionExtensionsTest < Inferno::Test
       Inferno checks that the payer server response has appropriate extensions and references to libraries within
       those extensions.
     )
+    verifies_requirements 'hl7.fhir.us.davinci-dtr_2.0.1@62', 'hl7.fhir.us.davinci-dtr_2.0.1@83',
```

Comment:
requirement @57 is tested through demonstrating pre-population extensions - if they can show they can do it, then that’s enough to satisfy the “where possible” I think since where possible is very subjective.

Reply:
Addressed in [6653e49](https://github.com/inferno-framework/davinci-dtr-test-kit/pull/59/commits/6653e495e069efcc7e8474e6becffb5493393339)